HousingSearchApi/V1/Factories/QueryablePerson.cs

```diff
@@ -0,0 +1,113 @@
+using System.Collections.Generic;
+using HousingSearchApi.V1.Domain;
+using Nest;
+using Identification = HousingSearchApi.V1.Domain.ElasticSearch.Identification;
+using Tenures = HousingSearchApi.V1.Domain.ElasticSearch.Tenures;
+
+namespace HousingSearchApi.V1.Factories
+{
+    public class QueryablePerson
+    {
+        public Person Create()
```

Comment:
What does this method do that `new Person() {Id = ..., Title = ...}` doesn't?

Should you not pass in parameters to the `Create()` method instead?

Reply:
Just convenience.  No need for parameters, it's a mapping function that spits out something else. Sometimes these can go into factories, sometimes I just leave them in a class.